### PR TITLE
[symlink] Support Jinja2 templating for "to" field

### PR DESCRIPTION
**Pull request by dolohow**
This is on par with "decompress" plugin.

### Motivation for changes:
I would like to inject data into path while making hard links.

### Config usage if relevant (new plugin or updated schema):
```
symlink:
      to: '/home/user/{{ variable }}'
```

Tested ✅

### CODE REVIEW

**Comment by dolohow** [comment]
Any plans on merging?

@liiight

File: flexget/plugins/output/symlink.py [hunk 1]
```diff
@@ -50,7 +51,12 @@ def on_task_output(self, task, config):
                 continue
             lnkfrom = entry['location']
             name = os.path.basename(lnkfrom)
-            lnkto = os.path.join(config['to'], name)
+            try:
+                lnkto = render_from_entry(os.path.join(config['to'], name), entry)
+            except RenderError as error:
+                log.error('Could not render path: %s', to)
```
Comment (cvium) [inline flexget/plugins/output/symlink.py]: What is `to` in this context? Doesn't look like it's initialized anywhere.

Comment (dolohow) [reply flexget/plugins/output/symlink.py]: Yes, it should be `lnkto`.  Will fix it in few hours when I get back home.

Nice catch.

Comment (dolohow) [reply flexget/plugins/output/symlink.py]: Fixed

**Comment by dolohow** [comment]
Is it good now? @liiight @cvium

**Pull request merged**